test/hotspot/jtreg/runtime/ErrorHandling/TimeoutInErrorHandlingTest.java

```diff
@@ -26,14 +27,17 @@
 import java.io.File;
 import java.io.FileInputStream;
 import java.io.InputStreamReader;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
 import java.util.regex.Pattern;
 
 import jdk.test.lib.process.OutputAnalyzer;
 import jdk.test.lib.Platform;
 import jdk.test.lib.process.ProcessTools;
 
 /*
- * @test
+ * @test TimeoutInErrorHandlingTest-default
```

Comment:
Isn't the better way to name this just '@test TimeoutInErrorHandlingTest' and then '@id=default' and '@id=with-on-error' ?

Reply:
Not sure what you mean? There is no @id jtreg tag. https://openjdk.org/jtreg/tag-spec.html

Reply:
> Not sure what you mean? There is no @id jtreg tag. https://openjdk.org/jtreg/tag-spec.html

Sorry, I got this wrong. The way to do it is @test id=$somestring. See for example:

https://github.com/openjdk/jdk/blob/a06426a52f16c08c95b1c0270a5fc40721921022/test/hotspot/jtreg/gc/stress/gcold/TestGCOldWithShenandoah.java#L27

Reply:
Oh, you are right. Its also plain wrong, that specifier does nothing and gets ignored. I'll switch to "id"